models/exposure.py

```diff
@@ -115,6 +115,31 @@ def clear_cache(self):
 im_type_enum = Enum("science", "reference", "difference", "bias", "dark", "flat", name='image_type')
 
 
+def get_short_image_type(image_type):
+    """
+    Get the short version of the image type.
+    If there is a short version, it will be returned,
+    otherwise returns the image_type unchanged.
+    Generally, longer image types will be shortened,
+    e.g., science -> sci, reference -> ref, difference -> sub.
+
+    Parameters
+    ----------
+    image_type: str or None
+
+    Returns
+    -------
+    str or None:
+         The short version of the image type, or None if the input was None.
+    """
+    short_types = {
+        "science": "sci",
```

Comment:
We probably also want to distinguish between individual science images and science stacks (e.g. if we want to do an end-of-night stack subtraction for when the same field is imaged multiple times in one night, or in one week, or whatever).